Microsoft.Toolkit.Uwp.UI.Controls/Carousel/Carousel.cs

```diff
@@ -234,6 +234,11 @@ private void FocusContainerFromIndex(int index)
         }
 
         /// <summary>
+        /// Occurs when the selected item has changed
+        /// </summary>
+        public event EventHandler<object> SelectionChanged = null;
```

Comment:
The AddedItems should be the newly selected item and the RemovedItems should be the old selected item. This is standard behavior for any control that allows for a selected item. They have the benefit of extending Selector which we can't